packages/components/CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
-- `Divider`, `Button Group`, `Carousel` and `Text List` match new design language
+- `Divider`, `Button Group`, `Carousel`, `Text List` and  `Link Tile` match new design language
```

Comment:
Wrong location now 😇

Reply:
Jesus, didn't catch that there was already a new release... 😬